modules/azure/src/main/scala/com.snowplowanalytics.snowplow.lakes/AuthenticationError.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2014-present Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This software is made available by Snowplow Analytics, Ltd.,
+ * under the terms of the Snowplow Limited Use License Agreement, Version 1.0
+ * located at https://docs.snowplow.io/limited-use-license-1.0
+ * BY INSTALLING, DOWNLOADING, ACCESSING, USING OR DISTRIBUTING ANY PORTION
+ * OF THE SOFTWARE, YOU AGREE TO THE TERMS OF SUCH LICENSE AGREEMENT.
+ */
+
+package com.snowplowanalytics.snowplow.lakes
+
+import org.apache.hadoop.fs.azurebfs.contracts.exceptions.AbfsRestOperationException
+
+import cats.implicits._
+
+object AuthenticationError {
+
+  private val knownErrors = Map(
+    90002 -> "Tenant is invalid. Make sure you have the correct tenant ID and are signing into the correct cloud. This may happen if there are no active subscriptions for the tenant",
```

Comment:
I think this part of the message...

> Make sure you have the correct tenant ID and are signing into the correct cloud. This may happen if there are no active subscriptions for the tenant

....is not phrased to be a helpful message for BDP customers.  I know you copied this from the original error message.  But I think it's the loader's job to just say "tenant is invalid", and then the console's job to wrap it by saying "here is the message from the loader, now go check the original setup instructions because you did it wrong".

tldr: I'd cut it to the first sentence only.

Reply:
I kept `Tenant is invalid or does not have an active subscription`